book/background_task.md

```diff
@@ -1,34 +1,77 @@
-# Background Tasks with Nu
+# Background Jobs
 
-Currently, Nushell doesn't have built-in background task management feature, but you can make it "support" background task with some tools, here are some examples:
+## Spawning Jobs
 
-1. Using a third-party task management tools, like [pueue](https://github.com/Nukesor/pueue)
-2. Using a terminal multiplexer, like [tmux](https://github.com/tmux/tmux/wiki) or [zellij](https://zellij.dev/)
+Nushell currently presents experimental support for thread-based background jobs.
 
-## Using Nu With Pueue
+Jobs can be can be spawned using [`job spawn`](/commands/docs/job_spawn.md), which receives a closure and starts its execution in a background thread, returning
+an unique integer id for such job:
 
-The module borrows the power of [pueue](https://github.com/Nukesor/pueue), it is possible to schedule background tasks to pueue, and manage those tasks (such as viewing logs, killing tasks, or getting the running status of all tasks, creating groups, pausing tasks etc etc)
+```nu
+'i am' | save status.txt
 
-Unlike terminal multiplexer, you don't need to attach to multiple tmux sessions, and get task status easily.
+job spawn { sleep 10sec; ' inevitable' | save --append status.txt }
+## => 1
 
-Here we provide a [nushell module](https://github.com/nushell/nu_scripts/tree/main/modules/background_task) that makes working with pueue easier.
+open status.txt
+## => i am
 
-Here is a setup example to make Nushell "support" background tasks:
+# wait for 10 seconds
+sleep 10sec
 
-1. Install pueue
-2. run `pueued`, you can refer to [start-the-daemon page](https://github.com/Nukesor/pueue/wiki/Get-started#start-the-daemon) for more information.
-3. Put the [task.nu](https://github.com/nushell/nu_scripts/blob/main/modules/background_task/task.nu) file under `$env.NU_LIB_DIRS`.
-4. Add a line to the `$nu.config-path` file: `use task.nu`
-5. Restart Nushell.
+open status.txt
+## => i am inevitable
+```
 
-Then you will get some commands to schedule background tasks. (e.g: `task spawn`, `task status`, `task log`)
+::: tip Note
+Unlike many other shells, Nushell jobs are **not** separate processes, and are instead implemented
+as background threads. An important side effect of that, is that all background jobs terminate once the shell process exits.
+:::
```

Comment:
could you expand this into a proper (but brief) section? things I think we should specifically cover:

- interactive vs. non-interactive job killing behavior on Nushell exit
- explain that this is the rationale behind no `disown` / eventually `job dispatch`